test/integ/quote-classic.test.ts

```diff
@@ -2573,6 +2576,10 @@ describe('quote', function () {
                 enableUniversalRouter: true,
               },
             ],
+            // actual classic-only quote doesn't pass in swapper,
```

Comment:
hm - you mean on the interface?

Reply:
yep I meant on the interface and mobile. I guess external api passes in swapper by default?